browser/pages/welcome/controller.js

```diff
@@ -2,24 +2,56 @@
 
 class WelcomeController {
 
-  constructor($state, $scope, electron) {
+  constructor($state, $scope, electron, request) {
     this.router = $state;
+    this.http = request;
     this.scope = $scope;
     this.scope.background = true;
     this.electron = electron;
-    $scope.version = electron.remote.app.getVersion();
+    this.scope.version = electron.remote.app.getVersion();
+    $scope.$watch('$viewContentLoaded', this.check.bind(this));
   }
 
   next() {
     this.router.go('location');
   }
 
+  check() {
+    let versionUrl = 'https://developers.redhat.com/download-manager/rest/available/devsuite/?nv=1';
+    this.scope.status = 'Checking'
+    let req = {
+      method: 'GET',
+      url: versionUrl
+    };
+
+    this.http(req).then((data)=>{
+      let version = this.scope.version;
+      let devsuiteVersion = data.data[0].featuredArtifact.versionName;
+      let numaricVersion = version.split('-')[0].replace(/\./g,'');
+      let numaricDevsuiteVersion = devsuiteVersion.split('-')[0].replace(/\./g,'');
+      if (numaricDevsuiteVersion.replace(/\./g,'') > numaricVersion.replace(/\./g,'')){
+        this.scope.textversion = 'Updated version:'
+        this.scope.newversion = devsuiteVersion;
+        this.scope.status = `New`;
+      } else {
+          this.scope.status = 'Current';
+      }
+    }).then(()=>{
+      this.scope.$apply();
+    });
```

Comment:
This could use some error handling, so it wouldn't just get stuck and spew errors when unable to connect etc.

And some tests please.